From 96c1d5c7b66b47f72263ee796190a1fbf50cc462 Mon Sep 17 00:00:00 2001
From: Andrei Gherzan <andrei@balena.io>
Date: Wed, 17 Jul 2019 15:34:18 +0100
Subject: rpi: Add rpi_model entry for RaspberryPi 4

The Raspebrry Pi 4 uses the new revision code scheme as documented by
the foundation[1]. This change adds an entry for this board as well.

[1] https://www.raspberrypi.org/documentation/hardware/raspberrypi/revision-codes/README.md

Signed-off-by: Andrei Gherzan <andrei@balena.io>
---
 board/raspberrypi/rpi/rpi.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/board/raspberrypi/rpi/rpi.c b/board/raspberrypi/rpi/rpi.c
index 153a1fdcb7..8b6278630f 100644
--- a/board/raspberrypi/rpi/rpi.c
+++ b/board/raspberrypi/rpi/rpi.c
@@ -148,6 +148,11 @@ static const struct rpi_model rpi_models_new_scheme[] = {
 		DTB_DIR "bcm2837-rpi-3-b-plus.dtb",
 		true,
 	},
+	[0x11] = {
+		"4 Model B",
+		DTB_DIR "bcm2711-rpi-4-b.dtb",
+		true,
+	},
 };
 
 static const struct rpi_model rpi_models_old_scheme[] = {
-- 
2.17.1

